src/guides/Filament-Manager-Installation.md

```diff
@@ -0,0 +1,122 @@
+# Setup guide for remote database using the plugin FilamentManager
+
+## Installation of PostgreSQL database on OctoFarm Client
+
+#### If you have filament records loaded into filament manager already on one of your OctoPrint instances please export them to save you loosing them
+
+On the OctoFarm client open the terminal and enter the following:
+
+Install the postgresql package:
+```
+$ sudo apt install postgresql
+```
+
+Switch to the postgres user:
+```
+$ sudo -u postgres -i
+```
+
+Create user octoprint:
+```
+[postgres]$ createuser --interactive -P
+
+Enter name of role to add: octoprint
+Enter password for new role:
+Enter it again:
+Shall the new role be a superuser? (y/n) n
+Shall the new role be allowed to create databases? (y/n) n
+Shall the new role be allowed to create more new roles? (y/n) n
+```
+
+Create database octoprint_filamentmanager:
+```
+[postgres]$ createdb -O octoprint octoprint_filamentmanager
+```
+
+Configure PostgreSQL for local network
+Per default PostgreSQL only accepts connections from localhost. To make the database server available in your network you have to modify two config files.
+```
+sudo nano /etc/postgresql/12/main/postgresql.conf
+```
+
+In /etc/postgresql/12/main/postgresql.conf change the line with listen_addresses to
+```
+listen_addresses = '*'
+```
+
+Then append the following line to /etc/postgresql/12/main/pg_hba.conf
+```
+sudo nano /etc/postgresql/12/main/pg_hba.conf
+```
+Adapt the IP address to your network, e.g. if your server has the IP 192.168.0.25 use 192.168.0.0/24 instead. This allows all clients in your network to access the database. For more information see https://en.wikipedia.org/wiki/Subnetwork
```

Comment:
You can make the wiki a clickable link :)

` [Cost Estimation](https://github.com/OllisGit/OctoPrint-CostEstimation)`